### ISSUE TITLE
Detection of handwritten characters

### ISSUE THREAD

**Issue by pratishthaa** [Niketkumardheeryan/ML-CaPsule#281]
Detect handwritten characters and deploy the project via a website or app.

**Comment by mo1998** [comment]
@pratishthaa 
Thank you.

**Comment by mo1998** [comment]
@pratishthaa 
I have been finished training the model and saved it.
Now i am working on building a website using flask and html to deploy it.
Is there any deadlines?

**Comment by pratishthaa** [comment]
5 days is the deadline from when you got assigned

**Comment by mo1998** [comment]
> 5 days is the deadline from when you got assigned

Thanks for information.

**Comment by heymansi** [comment]
Hii, @Rutikab12 could you please assign me this issue

**Comment by ishaanv1206** [comment]
Hi @pratishthaa 
I would like to work over it and deploy the model on a web application. Please assign the issue to me.

**Comment by Likhita4403** [comment]
Hi. I am Likhita, a pariticipant of GSSOC'24.
I plan to follow the below methodology to achieve the objective to develop a robust model.
Methodology:
1. Utilize datasets such as MNIST for digits and EMNIST for extended characters, supplemented by data augmentation techniques like rotations and noise addition.

2. Experiment with various neural network architectures including Convolutional Neural Networks (CNNs) for image processing and explore the potential of transformers adapted for image recognition tasks.

3. Perform hyperparameter tuning and validate using k-fold cross-validation to ensure model robustness.

4. Measure performance through metrics such as accuracy, precision, recall, and analyze with a confusion matrix.

I would like to work on this issue, I have previously worked on visual transformers and NLP related tasks previously.

**Comment by rohanth3** [comment]
hi @Rutikab12 or @pratishthaa i would like to work on this issue can you assign me I am a GSSoC 2026 contributor on the Open Source + AI Agents track

**Comment by coolenough** [comment]
@pratishthaa Hi I am GSSoC 2026 contributor I have prior experience of working on such type of tasks, I would like get this assigned to me

**Comment by nensii21** [comment]
Hi @pratishthaa , I am a GSSoC'26 contributor and would like to work on this issue. I have experience with machine learning, computer vision, and web deployment. If this issue is still open for contributions, kindly assign it to me.
